src/services/prisma-datastore/db-init.ts

```diff
@@ -18,436 +34,459 @@ const prisma = new PrismaClient()
+    valueType: ReadingSchemaValueType.SCALAR_FLOAT,
+    unitSymbol: "V",
+    unit: "Voltage",
+    spec: {
+      voltage: 3.3,
     },
-    {
-        name: NodeSensorTypeNames.LORA_SIGNAL_STRENGTH,
-        measure: "rssi_lora",
-        uuid: "5c59bd9d-b56e-4faf-bfde-2de836cf39d9",
-        displayName: referenceSensorName("RSSI"),
-        displayMeasure: "Signal Strength",
-        eventName: nodeEvent(sensors.db),
-        valueType: ReadingSchemaValueType.SCALAR_INT,
-        unitSymbol: "dBm",
-        unit: "decibel milliwatts",
-        spec: {
-            sensor_rssi: 1
-        }
+  },
+  {
+    name: NodeSensorTypeNames.LORA_SIGNAL_STRENGTH,
+    measure: "rssi_lora",
+    uuid: "5c59bd9d-b56e-4faf-bfde-2de836cf39d9",
+    displayName: referenceSensorName("RSSI"),
+    displayMeasure: "Signal Strength",
+    eventName: nodeEvent(sensors.db),
+    valueType: ReadingSchemaValueType.SCALAR_INT,
+    unitSymbol: "dBm",
+    unit: "decibel milliwatts",
+    spec: {
+      sensor_rssi: 1,
     },
-    {
-        name: GatewaySensorTypeNames.LORA_SIGNAL_STRENGTH,
-        measure: "rssi_lora",
-        uuid: "591305a3-6b51-4f1f-a2fb-fd685fdc3fdd",
-        displayName: GatewaySensorTypeNames.LORA_SIGNAL_STRENGTH,
-        displayMeasure: "Signal Strength",
-        eventName: nodeEvent(sensors.db),
-        valueType: ReadingSchemaValueType.SCALAR_INT,
-        unit: "dBm",
-        spec: {
-            gateway_rssi: 1
-        }
-    }
+  },
+  {
+    name: GatewaySensorTypeNames.LORA_SIGNAL_STRENGTH,
+    measure: "rssi_lora",
+    uuid: "591305a3-6b51-4f1f-a2fb-fd685fdc3fdd",
+    displayName: GatewaySensorTypeNames.LORA_SIGNAL_STRENGTH,
+    displayMeasure: "Signal Strength",
+    eventName: nodeEvent(sensors.db),
+    valueType: ReadingSchemaValueType.SCALAR_INT,
+    unit: "dBm",
+    spec: {
+      gateway_rssi: 1,
+    },
+  },
 ];
 
 /**
- * Creates a new project. 
- * @param projectUID 
+ * Creates a new project.
+ * @param projectUID
  */
 async function createTypicalProject(prisma: PrismaClient, projectUID: string) {
-    const gateways = [
-        { 
-            deviceUID: 'dev:fake',
-            name: 'fake.name',
-            location: 'fake.location',
-            nodes: [
-                {
-                    nodeEUI: 'fake.deviceEUI.1',
-                    name: 'fake.name.1',
-                    location: 'fake.location.1'
-                },
-                {
-                    nodeEUI: 'fake.deviceEUI.2',
-                    name: 'fake.name.2',
-                    location: 'fake.location.2'
-                }
-            ]
-        }
-    ]
-   
-    const project = await upsertProject({ prisma, projectUID, name: "faker.something" });
-    const schemas = standardSchemas.map(readingSchemaDefaults);
-    await upsertReadingSchemas(prisma, project.readingSource, schemas);
-    await upsertGatewaysAndNodes(prisma, project, gateways);
-    return project;
+  const gateways = [
```

Comment:
Now that we have bulk import in place, we don't need the fake data. It was just for PoC/initial testing.